sdk/samples/iot/provisioning/README.md

```diff
@@ -56,57 +76,35 @@ All samples require either x509 certification or SAS symmetric key authenticatio
     ```cmd
     generate_certificate.cmd
     ```
-
-2. For Windows (or if required on your OS), set the environment variable `AZ_IOT_DEVICE_X509_TRUST_PEM_FILE` to the path of the BaltimoreCyberTrustRoot.crt.pem file noted near the bottom of the output. You must [download this pem file](https://cacerts.digicert.com/BaltimoreCyberTrustRoot.crt.pem) and store it in the location specified by that filepath.
-
-3. The remaining output will be used in the next steps.
-### #Paho IoT Provisioning (Certificates)
-**Executable:** paho_iot_provisioning_sample
-
-This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c) uses x509 authentication to connect to Azure IoT Hub Device Provisioning Service (DPS).
-1. Set the environment variable `AZ_IOT_DEVICE_X509_CERT_PEM_FILE` to the path of the generated .pem file noted at the bottom of the output.
-
 2. In your Azure DPS, add a new individual device enrollment using the recently generated `device_ec_cert.pem` file.  See [here](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-x509#create-a-device-enrollment-entry-in-the-portal) for further instruction.  After creation, the Registration ID of your device should appear as `paho-sample-device1` in the Individual Enrollments tab.
+3. Set the following environment variables:
+*  `AZ_IOT_DEVICE_X509_CERT_PEM_FILE`: Copy the path of the generated .pem file noted at the bottom of the generate_certificate output.
+*  `AZ_IOT_REGISTRATION_ID`: This should be `paho-sample-device1`.
+*  `AZ_IOT_ID_SCOPE`: Copy this value from the Overview tab in your Azure DPS.
 
 ### Paho IoT Provisioning (SAS)
-**Executable:** paho_iot_provisioning_sas_sample
+**Executable:** paho_iot_provisioning_sas_example
 
 This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/provisioning/src/paho_iot_provisioning_sas_example.c) uses SAS symmetric key authentication to connect to Azure IoT Hub Device Provisioning Service (DPS).
 
-* In your Azure DPS, add a new individual device enrollment using SAS. See [here](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#create-a-device-enrollment-entry-in-the-portal) for further instruction, with one exception--for the Primary Key, **you must use** the fingerprint noted in the output.  **Do NOT use** the *Auto-generate keys* option. After creation, the Registration ID of your device will appear in the Individual Enrollments tab.
-
-## Build and Run the Sample
-
-1. Set the remaining environment variables.  Setting a variable will take the following form:
+1. In your Azure DPS, add a new individual device enrollment using SAS. See [here](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#create-a-device-enrollment-entry-in-the-portal) for further instruction. After creation, the Registration ID of your device will appear in the Individual Enrollments tab.
```

Comment:
nit: Add back-ticks.
```suggestion
1. In your Azure DPS, add a new individual device enrollment using SAS. See [here](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#create-a-device-enrollment-entry-in-the-portal) for further instructions. After creation, the `Registration ID` of your device will appear in the `Individual Enrollments` tab.
```

Reply:
I disagree.  Those are titles of items you find in the browser, not something you would type into or read off the command line, and hence would also be distracting here.

Reply:
Fair enough :)

```suggestion
1. In your Azure DPS, add a new individual device enrollment using SAS. See [here](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#create-a-device-enrollment-entry-in-the-portal) for further instructions. After creation, the Registration ID of your device will appear in the Individual Enrollments tab.
```